Cargo.lock

```diff
@@ -1301,6 +1403,18 @@ dependencies = [
  "quick-error 1.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "ring"
+version = "0.8.1"
```

Comment:
Both travis-ci and I are getting build failures like:

```
          /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/ring-0.12.1/crypto/curve25519/curve25519.c:4693: multiple definition of `GFp_x25519_public_from_private'
          /home/travis/build/mozilla-services/syncstorage-rs/target/debug/deps/libring-ae014aa1941bdf5c.rlib(curve25519.o):/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/ring-0.8.1/crypto/curve25519/curve25519.c:4712: first defined here
```

Could be because this is introducing a second version of `ring`, and both versions can't be compiled together into a single binary?

Reply:
> Could be because this is introducing a second version of `ring`...

Looks like it, yep. Thanks for the tip, I'll make sure I sort it before de-wipping this.